common/src/main/resources/assets/adorn/lang/it_it.json

```
@@ -1,1278 +1,1373 @@
-{
-  "block.adorn.white_sofa": "Divano Bianco",
-  "block.adorn.light_gray_sofa": "Divano Grigio Chiaro",
-  "block.adorn.gray_sofa": "Divano Grigio",
-  "block.adorn.black_sofa": "Divano Nero",
-  "block.adorn.brown_sofa": "Divano Marrone",
-  "block.adorn.red_sofa": "Divano Rosso",
-  "block.adorn.orange_sofa": "Divano Arancione",
-  "block.adorn.yellow_sofa": "Divano Giallo",
-  "block.adorn.lime_sofa": "Divano Lime",
-  "block.adorn.green_sofa": "Divano Verde",
-  "block.adorn.cyan_sofa": "Divano Ciano",
-  "block.adorn.light_blue_sofa": "Divano Celeste",
+{acacia_chair
```

This probably doesn't belong here:
```suggestion
{
```

---

Yes sorry lol.